l10n_it_withholding_tax/models/account.py

```diff
@@ -52,9 +52,8 @@ def create(self, vals):
             ml_ids.append(vals.get("credit_move_id"))
         move_lines = self.env["account.move.line"].browse(ml_ids)
         invoice = move_lines.filtered(lambda x: x.exists()).move_id.filtered(
-            lambda x: x.is_invoice()
+            lambda x: x.withholding_tax
         )
```

Comment:
Il test su invoice.withholding_tax c'è già sotto.
Qui vanno escluse le account_move che non sono fatture, e bisogna in tal modo dovrebbe rimanere un solo record.

In realtà non fa differenza, il risultato non cambia, alla fine dentro l'if si entra se invoice esiste e ha .witholding_tax 
Tuttavia fallisce se - per qualche oscuro motivo - dentro vals finiscono account.move.line che non appartengono ad una fattura ma ad un altro movimento (che non dovrebbe avere comunque .withholding_tax).